spec/amp-3p-video.md

```diff
@@ -0,0 +1,82 @@
+# Vendor-specific (3p) video player components
+
+The generic [`amp-video`](https://go.amp.dev/c/amp-video) component plays videos directly on a page, much like a `<video>` tag.
+
+Some video cannot be embedded this way. AMP provides vendor-optimized components like [`amp-youtube`](https://go.amp.dev/c/amp-youtube),
```

Comment:
Perhaps we can reword this a bit....

> Unfortunately, not all videos can be embedded this way. For these specialized cases, AMP provides vendor-optimized components like [`amp-youtube`](https://go.amp.dev/c/amp-youtube), [`amp-ima-video`](https://go.amp.dev/c/amp-ima-video), and [others](./amp-video-interface.md).
Internally these players load an iframe whose page communicates with the outer document to coordinate playback.